v2.1/cancel-query.md

```diff
@@ -69,7 +70,7 @@ In this example, we nest a [`SELECT` clause](select-clause.html) that retrieves
           AND query = 'SELECT * FROM test.kv ORDER BY k');
 ~~~
 
-{{site.data.alerts.callout_info}}<code>CANCEL QUERY</code> accepts a single query ID. If subquery is used and returns multiple IDs, the <code>CANCEL QUERY</code> statement will therefore fail.{{site.data.alerts.end}}
+{{site.data.alerts.callout_info}}<code>CANCEL QUERY</code> accepts a single query ID. If a subquery is used and returns multiple IDs, the <code>CANCEL QUERY</code> statement will fail. To cancel multiple queries, use the <code>CANCEL QUERIES</code> syntax.{{site.data.alerts.end}}
```

Comment:
I might simplify to the shorter "To cancel multiple queries, use CANCEL QUERIES".